lib/ddtrace.rb

```diff
@@ -29,10 +29,13 @@ def self.tracer
 if defined?(Rails::VERSION)
   if Rails::VERSION::MAJOR.to_i >= 3
     require 'ddtrace/contrib/rails/framework'
+    require 'ddtrace/contrib/rails/middlewares'
 
     module Datadog
       # Railtie class initializes
       class Railtie < Rails::Railtie
+        config.app_middleware.use(Datadog::Contrib::Rails::TraceMiddleware)
```

Comment:
Ouch saw that it's `Rails` and not `Rack`. Can we call it ExceptionMiddleware or similar if it's handling only exceptions?